content/docs/studio/get-started.md

```diff
@@ -42,13 +42,21 @@ details]).
 
    </admon>
 
-3. Iterative Studio parses your project's `dvc.yaml` file to identify data,
-   metrics, plots and hyperparameters.
-
-   If you are not using DVC, you can separately
-   [indicate which files contain metrics and hyperparameters](/doc/studio/user-guide/projects-and-experiments/configure-a-project#custom-metrics-and-parameters).
-   However, we strongly recommend using DVC to avail of all the features of
-   Iterative Studio.
+3. Iterative Studio identifies experiments and their data, metrics, plots and
+   hyperparameters form the following sources:
+
+   1. **`dvc.yaml` file**: Details of experiments saved as Git commits are
+      fetched from this file.
+   2. **Git refs**: Details of experiments pushed using `dvc exp push` are
+      fetched from here.
+   3. **Metrics and plots logged by [DVCLive]**: To [track live metrics and
+      plots][live-metrics-and-plots] of running experiments, you should set the
+      `DVC_STUDIO_TOKEN` environment variable and use [DVCLive] in your training
+      pipeline.
+   4. **Files that you
+      [indicate as custom metrics and parameters files](/doc/studio/user-guide/projects-and-experiments/configure-a-project#custom-metrics-and-parameters)**:
+      These are useful if you are not using DVC. However, we strongly recommend
+      using DVC to avail of all the features of Iterative Studio.
```

Comment:
I think this section may be too complicated for the get started. Can we cut it down to something like this?

> Iterative Studio relies on Git commits as experiments and identifies data, metrics, plots and hyperparameters using [DVC experiment tracking](https://dvc.org/doc/start/experiments/experiment-tracking).

Reply:
Alternatively, I think what you had before these changes was fine. All of these sources except for custom metrics use `dvc.yaml`. Do we really need all this additional info? At most, I would mention DVCLive as a starting point.

Reply:
Thanks @dberenbaum. Agreed, it looks too complicated.
I've reverted to the previous version, with only a slight modification to mention Git refs. PTAL.